### ISSUE TITLE
"contentLoaded is not defined" Error

### ISSUE THREAD

**Issue by samwilliscreative** [closingtag/calc-polyfill#26]
I've got my script running after my CSS is pulled in, like it says to do in the documentation. However, I'm still getting this error:

```
contentLoaded is not defined
(anonymous function)    @   calc.js:511
(anonymous function)    @   calc.js:544
```

**Comment by rodebert** [comment]
Do you use the calc.js from the dist folder? https://github.com/closingtag/calc-polyfill/blob/master/dist/calc.js

**Comment by samwilliscreative** [comment]
Yes, that's the code I used. I've just replaced the code on my version with the code from the file you linked to above, and I am still getting `contentLoaded is not defined`, but now without the lines that are causing the error.

If it helps, the project I'm working on is using asp .net, and the script file is injected in the code behind. See this stack overflow question for an example: http://stackoverflow.com/questions/3198123/proper-way-to-include-scripts-in-an-asp-net-page

`protected override void Render(HtmlTextWriter writer)
 {
    Page.ClientScript.RegisterClientScriptInclude("jQuery",        ResolveClientUrl("~/scripts/jquery/js/jquery-1.4.2.min.js"));    
    base.Render(writer);
}`
